backend/src/main/java/com/festago/festival/presentation/v1/FestivalV1Controller.java

```diff
@@ -0,0 +1,36 @@
+package com.festago.festival.presentation.v1;
+
+import com.festago.common.exception.BadRequestException;
+import com.festago.common.exception.dto.ErrorResponse;
+import com.festago.festival.application.FestivalV1QueryService;
+import com.festago.festival.dto.FestivalV1ListRequest;
+import com.festago.festival.dto.FestivalV1Response;
+import jakarta.servlet.http.HttpServletRequest;
+import lombok.RequiredArgsConstructor;
+import org.springframework.data.domain.Slice;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/api/v1/festivals")
+@RequiredArgsConstructor
+public class FestivalV1Controller {
+
+    private final FestivalV1QueryService festivalV1QueryService;
+
+    @GetMapping
+    public ResponseEntity<Slice<FestivalV1Response>> findFestivals(
+        FestivalV1ListRequest festivalListRequest) {
+        return ResponseEntity.ok(festivalV1QueryService.findFestivals(festivalListRequest));
+    }
```

Comment:
```suggestion
    @GetMapping
    public ResponseEntity<Slice<FestivalV1Response>> findFestivals(
        @PageableDefault(size = 10) Pageable pageable,
        FestivalV1ListRequest request
    ) {
        return ResponseEntity.ok(festivalV1QueryService.findFestivals(pageable, request));
    }
```
다음과 같이 `Pageable`을 받는건 어떤가요?

```java
public class FestivalV1QueryService {
    ...
    public Slice<FestivalV1Response> findFestivals(Pageable pageable, FestivalV1ListRequest request) {
        return festivalRepository.findBy(
            new FestivalSearchCondition(
                request.filter(),
                request.region(),
                request.lastStartDate(),
                request.lastFestivalId(),
                pageable,
                LocalDate.now(clock)
            )
        );
    }
    ...
}
```
그렇게 된다면 DTO가 조금 덜 똑똑해질 것 같네요!

Reply:
이러면 request 명세가 바뀔 것 같아서 우선 현 상태를 유지하겠습니다!!